shared/components/ResourceGraph/buildGraph/buildNetworkGraph.js

```diff
@@ -139,15 +146,21 @@ export function buildNetworkGraph({ store }, config) {
           const podId = multiplePods
             ? 'composite-pod'
             : store['Pod'][0].metadata.uid;
-          if (!!deployment && store['Pod']?.length) {
-            currentLayer.forEach(svc => {
+          const subscriptionId = store['Subscription'].map(
```

Comment:
I'd name it `subscriptionIds` (with s) to clearly show that it's a list. Also, I would add a question mark `?` before .map, we all know it sometimes likes to crash 🙃